docs2/site/docs/getting-started/schema-types.md

```diff
@@ -82,9 +86,32 @@ enum Episode {
 }
 ```
 
+**.NET**
+
+
+Consider the equivalent `enum` in .NET:
+
+```csharp
+public enum Episodes
+{
+    NEWHOPE  = 4,
+    EMPIRE  = 5,
+    JEDI  = 6
+}
+```
+
+Compare the two implementations. GraphQL does not specify backing values for members of its enums. The name of each member _is_ the value.
+
 **GraphQL .NET**
 
-You can manually create the `EnumerationGraphType`.
+GraphQL.NET provides two methods of defining GraphQL enums.
+
+You can manually create the `EnumerationGraphType`. Advantages of this method:
+
+- The GraphQL enum need not map to a specific .NET `enum`. You could, for instance, build the enum from one of the alternate methods of defining discrete sets of values in .NET, such as classes of constants or static properties
+- Allows descriptions of enum members
```

Comment:
Obsolete. `EnumerationGraphType<TEnum>` sets descriptions.

Reply:
I will add this detail but the existing method of defining enum value descriptions is not obsolete in the sense that it's still part of the public API and is not itself marked obsolete. Not everyone will be keen to use the attribute method of defining enum descriptions.

Reply:
I am sorry, I incorrectly expressed. I just wanted to say that the advantages of this method can be perceived as opposed to the generic version, which did not have such capabilities before (descriptions, deprecations). I agree that it's OK to say about the explicit possibilities of the first method. Then it is worth clarifying that the second method also supports similar capabilities, but through attributes.